cni-plugin/deployment/scripts/install-cni.sh

```diff
@@ -315,31 +299,39 @@ monitor() {
     done
 }
 
+log() {
+  printf '[%s] %s\n' "$(date '+%Y-%m-%d %H:%M:%S')" "$1"
+}
+
 ################################
 ### CNI Plugin Install Logic ###
 ################################
 
+# Delete old "interface mode" file
+rm -f "${DEFAULT_CNI_CONF_PATH}"
```

Comment:
Is this for backwards compatibility? i.e. to clean up unused files? if so, it should automatically be cleaned up when the pod is restarted so it might not be necessary?

Reply:
Yes I had BC in mind. Isn't this the host filesystem, so the pod restart won't touch that? Or you mean it should be cleaned up by this plugin's logic?

Reply:
If the pod shuts down uncleanly that file will be left around, so we'd still have to keep some logic around to clean it up at startup, right? I'll add a comment to remember to remove that `rm` call at some point :+1: